models/public/mobilefacedet-v1-mxnet/mobilefacedet-v1-mxnet.md

```diff
@@ -0,0 +1,104 @@
+
+    WARNING: Note that input layout of converted model is `[BxHxWxC]`.
+
+## Output
+
+### Original model
+
+1. The array of detection summary info, name - `yolov30_slice_axis1`,  shape - `1,18,8,8`. The anchor values are `118,157,  186,248,  285,379`.
+
+2. The array of detection summary info, name - `yolov30_slice_axis2`,  shape - `1,18,16,16`. The anchor values are `43,54,  60,75,  80,106`.
+
+3. The array of detection summary info, name - `yolov30_slice_axis3`,  shape - `1,18,32,32`. The anchor values are `10,12,  16,20,  23,29`.
+
+For each case format is `B,N*DB,Cx,Cy`, where
+    - `B` - batch size
+    - `N` - number of detection boxes for cell
+    - `DB` - size of each detection box
+    - `Cx`, `Cy` - cell index
+
+Detection box has format [`x`,`y`,`h`,`w`,`conf`,`class`], where:
+- (`x`,`y`) - raw coordinates of box center, apply [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get coordinates relative to the cell
+- `h`,`w` - raw height and width of box, apply [exponential function](https://en.wikipedia.org/wiki/Exponential_function) and multiply by corresponding anchors to get height and width values relative to cell
+- `conf` - confidence of detection box, use [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get confidence in [0,1] range
+- `class` - probability that detected object belongs to specific class, use [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get confidence in [0,1] range
+
+### Converted model
+
+1. The array of detection summary info, name - `yolov30_yolooutputv30_conv0_fwd/YoloRegion`,  shape - `1,18,8,8`. The anchor values are `118,157,  186,248,  285,379`.
+
+2. The array of detection summary info, name - `yolov30_yolooutputv31_conv0_fwd/YoloRegion`,  shape - `1,18,16,16`. The anchor values are `43,54,  60,75,  80,106`.
+
+3. The array of detection summary info, name - `yolov30_yolooutputv32_conv0_fwd/YoloRegion`,  shape - `1,18,32,32`. The anchor values are `10,12,  16,20,  23,29`.
+
+For each case format is `B,N*DB,Cx,Cy`, where
+    - `B` - batch size
+    - `N` - number of detection boxes for cell
+    - `DB` - size of each detection box
+    - `Cx`, `Cy` - cell index
+
+Detection box has format [`x`,`y`,`h`,`w`,`conf`,`class`], where:
+- (`x`,`y`) - raw coordinates of box center, apply [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get coordinates relative to the cell
+- `h`,`w` - raw height and width of box, apply [exponential function](https://en.wikipedia.org/wiki/Exponential_function) and multiply by corresponding anchors to get height and width values relative to cell
+- `conf` - confidence of detection box, use [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get confidence in [0,1] range
+- `class` - probability that detected object belongs to specific class, use [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get confidence in [0,1] range
```

Comment:
Are you sure, that sigmoid is necessary for **converted** model? AFAIK, `TFYOLOV3` transformations already apply some postprocessing.